meadow/app/Webserver.hs

```diff
@@ -79,9 +83,17 @@ app handlers _staticDir githubEndpoints config =
       simpleCorsResourcePolicy
         {corsRequestHeaders = ["content-type", "set-cookie"]}
 
+startStatsd :: MonadIO m => m Store
+startStatsd = liftIO $ do
+    store <- newStore
+    void $ forkStatsd defaultStatsdOptions store
+    pure store
```

Comment:
Do we need to close this on shutdown, or is it a child process that will die gracefully?

Reply:
according to the documentation there is no need to do anything